repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,683 @@
+        hippotaxonomy:key: quality-and-improvement-outcomes
+        locale.en.name: Quality and Improvement Outcomes
+      /regional-skills-development-networks:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: regional-skills-development-networks
+        locale.en.name: Regional Skills Development Networks
+      /repositories:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: repositories
+        locale.en.name: Repositories
+      /research:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: research
+        locale.en.name: Research
+      /resource-discovery:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: resource-discovery
+        locale.en.name: Resource Discovery
+      /school-leavers:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: school-leavers
+        locale.en.name: School leavers
+      /service-delivery:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: service-delivery
+        locale.en.name: Service delivery
+      /social-care:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: social-care
+        locale.en.name: Social care
+      /statistics-and-data:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: statistics-and-data
+        locale.en.name: Statistics and data
+      /strategy:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: strategy
+        locale.en.name: Strategy
+      /streamlining:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: streamlining
+        locale.en.name: Streamlining
+      /sustainability:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: sustainability
+        locale.en.name: Sustainability
+      /tools-and-techniques:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: tools-and-techniques
+        locale.en.name: Tools and techniques
+      /training:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: training
+        locale.en.name: Training
+      /use-of-technology:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: use-of-technology
+        locale.en.name: Use of technology
+      /user-experience:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: user-experience
+        locale.en.name: User experience
+      /wellbeing:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: wellbeing
+        locale.en.name: Wellbeing
+      /wider-profession:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: wider-profession
+        locale.en.name: Wider profession
+      /workforce:
+        jcr:primaryType: hippotaxonomy:category
+        hippotaxonomy:key: workforce
+        locale.en.name: Workforce
+    /GlobalTags[3]:
```

Comment:
Only the `advocacy` category seems to have been published. Could we publish the remaining ones as well unless it was intentional ?

Reply:
Now resolved